.vscode/suggested.settings.json

```diff
@@ -0,0 +1,29 @@
+{
+	"[html]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+	"[javascript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+	"[json]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+	"[markdown]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+	"[scss]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+	"[typescript]": {
+		"editor.defaultFormatter": "esbenp.prettier-vscode"
+	 },
+	"[vue]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+	"editor.formatOnSave": true,
+	"search.exclude": {
+    "node_modules": true,
+    "dist": true,
+		"package-lock.json": true
+  }
```

Comment:
tabs or spaces? ;-)

Reply:
Formatting settings are set in Prettier: https://github.com/n8n-io/n8n/blob/02549e3ba9233a6d9f75fc1f9ff138e2aff7f4b9/.prettierrc.js#L20

Reply:
Why is this file inconsistently formatted then?

Reply:
Because this PR was meant to be for VSCode settings only. Should formatting for files outside of packages (this one and all others) be in this PR or separate?